docs/src/main/sphinx/connector/base-jdbc.rst

```diff
@@ -0,0 +1,194 @@
+as user facing features.
+
+.. _base-jdbc-table-statistics:
+
+Table statistics
+^^^^^^^^^^^^^^^^
+
+Connectors supports :doc:`table and column statistics </optimizer/statistics>`.
+They are used for :doc:`cost based optimizations
+</optimizer/cost-based-optimizations>` to improve query processing performance
+based on the actual data in the data source.
+
+Collection of statistics varies for each data source.
+
+Table and column statistics are enabled by default. You can disable it for a
+catalog by setting ``statistics.enabled`` to ``false``. The ``metadata``
+properties from the preceding section further control the statistics caching.
+
+The statistics can be viewed in Trino using :doc:`/sql/show-stats`.
+
+.. _base-jdbc-join-pushdown:
+
+Join pushdown
+^^^^^^^^^^^^^
+
+Join pushdown allows the connector delegate the table join operation to the
+underlying data source. This can result in performance gains and allows Trino to
+perform the remaining query processing on a smaller amount of data.
+
+The specifics for the supported pushdown of table joins varies for each data
+source and therefore for each connector.
+
+.. _base-jdbc-topn-pushdown:
+
+TopN pushdown
+^^^^^^^^^^^^^
+
+TopN pushdown, also called limit pushdown, refers to the common SQL clauses for
+``SELECT`` statements ``TOP N`` or ``LIMIT N``. These equivalent clauses reduce
+the number of rows returned by a query to ``N`` rows. A pushdown of this clause
+to the underlying data source can improve the performance of the query and
+significantly reduce the amount of data transferred from the data source to
+Trino.
+
+Implementation and support is connector specific since different data sources
+support different SQL syntax.
+
+.. _base-jdbc-aggregation-pushdown:
+
+Aggregation pushdown
+^^^^^^^^^^^^^^^^^^^^
+
+Some :doc:`aggregation functions can be pushed down </optimizer/pushdown>` to
+the data source for processing. This can result in significant performance gain.
+Each data source supports different aggregation functions, and pushdown is
+therefore different for each connector.
+
+.. _base-jdbc-connection-pooling:
+
+JDBC connection pooling
+^^^^^^^^^^^^^^^^^^^^^^^
+
+JDBC connection pooling is disabled by default. When enabled each worker creates
+and maintains a connection pool instead of opening and closing separate
+connections to the data source sequentially. Each connection is available to
+connect to the data source and retrieve data. After completion of the operation,
+the connection is returned to the pool and can be reused. This improves
+performance by a small amount, and reduces the load on any required
+authentication system used for establishing the connection.
+
+You can enable JDBC connection pooling by setting
+``connection-pool.enabled=true`` in your catalog properties file.
+
+Additional configuration properties to tune the connection pooling are listed
+below with their default value.
+
+.. code-block:: properties
+
+    connection-pool.enabled=true
```

Comment:
there is no such config

Reply:
See above. Connection pooling is likely to turn out to be Starburst only.

Reply:
It's starburst only, except oracle. somewhere in this PR I posted a screencap...